scrapy/http/cookies.py

```diff
@@ -170,7 +170,8 @@ def has_header(self, name):
         return name in self.request.headers
 
     def get_header(self, name, default=None):
-        return to_unicode(self.request.headers.get(name, default), errors="replace")
+        value = self.request.headers.get(name, default)
+        return to_unicode(value, errors="replace") if value else value
```

Comment:
I think it will return `''` where we want to return `b''` :)

Reply:
You are right, I fix it and add a test